code/modules/uplink/uplink_items.dm

```diff
@@ -1235,6 +1235,23 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	cost = 8
 	exclude_modes = list(/datum/game_mode/nuclear) //you can't buy it in nuke, because the elite hardsuit costs the same while being better
 
+datum/uplink_item/suits/hardsuit/cybersun
+	name = "Cybersun Hardsuit"
+	desc = "The long forgotten hardsuit made by Cybersun industries. \
+			Offers ROBUST protection against laser-based weapons, while still giving somewhat good chances \
+			to survive assault with toolbox or shotgun. \
+			Not to mention, it doesn't slow you down and contains integrated jetpack that runs on standard tanks. \
+			Systems in this hardsuit make it really hard to take it off from you."
+	item = /obj/item/clothing/suit/space/hardsuit/cybersun
+	cost = 14
+	exclude_modes = list(/datum/game_mode/nuclear) //nuke ops get it for lower value
+
+datum/uplink_item/suits/hardsuit/cybersun/nuke
+	item = /obj/item/clothing/suit/space/hardsuit/cybersun
```

Comment:
```suggestion
```

Reply:
you dont need this its inherited

Reply:
Even the item is inherited? Interesting.